viewer/views/layout.jade

```diff
@@ -154,6 +154,14 @@ html
         - if (molochClusters)
           each key in Object.keys(molochClusters)
             command.sendSessionButton(cluster=key,label="Send Session to #{molochClusters[key].name}")
+        - if (titleLink == "sessionsLink")
+          span 
+            b Reports
```

Comment:
Good idea, but since the type of the fragment lies in the object itself (and JS currently has no predicate/LINQ-like query method to check whether the collection contains an object with certain property with a certain value) I could do separate collections for query fragments and session fragments, so it would make the checking on the template easier. Question is, is it worth it?